src/ng/sce.js

```diff
@@ -413,7 +413,10 @@ function $SceDelegateProvider() {
  * Here's an example of a binding in a privileged context:
  *
  * ```
- * <input ng-model="userHtml">
+ * <label>
+ *   User input
```

Comment:
This seems to be changing the UI by adding additional content. Seems like that should be fine, but thought I'd call it out since so far that has not been the case.